src/Symfony/Component/Notifier/Tests/Channel/AbstractChannelTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Notifier\Tests\Channel;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Channel\AbstractChannel;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Notification\Notification;
+use Symfony\Component\Notifier\Recipient\Recipient;
+
+/**
+ * @author Jan Schädlich <jan.schaedlich@sensiolabs.de>
+ */
+class AbstractChannelTest extends TestCase
+{
+    public function test_a_channel_cannot_be_constructed_without_transport_and_bus(): void
```

Comment:
we use camel-case
and `void` should be removed: it means nothing on test cases - they're pure noise